README.md

```diff
@@ -1,70 +1,46 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Europe-Quick-Check App
 
-## Available Scripts
+> Europe-Quick-Check App  allows you to:
 
-In the project directory, you can run:
+- Display a list of European countries
+- Search any European country of your choice
+- Details of each Country.
 
-### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Built With
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- React and Redux
 
-### `npm test`
+## Live Demo 
+Feel free to check the [Live link](https://europe-quick-check.herokuapp.com/).
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Live Video
+[video link](https://drive.google.com/file/d/1iAn6DcECNdvFErQ5Kc5n7IdQFC_kB393/view?usp=sharing).
 
-### `npm run build`
+## Author
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+👤 **Oyewunmi Gbenga**
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- GitHub: [@githubhandle](https://github.com/kelomo2502)
+- Twitter: [@twitterhandle](https://twitter.com/kelomoJs)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/oyewunmi-gbenga)
 
```

Comment:
- [ **OPTIONAL** ] **RECOMENDED:** I think it would be better if you add one important section in your README file, `## Getting Started` section(like in the **[Microverse README template](https://github.com/microverseinc/readme-template)**), where you can put some information on how to **clone the repo, access the cloned directory, Prerequisites tools, set up the project,...**